www-nginx/ngx-set-misc/ngx-set-misc-0.33.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_PN="set-misc-nginx-module"
+inherit nginx-module
+
+DESCRIPTION="An NGINX module that adds various set_xxx directives to NGINX's rewrite module"
+HOMEPAGE="https://github.com/openresty/set-misc-nginx-module"
+SRC_URI="
+	https://github.com/openresty/set-misc-nginx-module/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz
+
+"
+NGINX_MOD_S="${WORKDIR}/${MY_PN}-${PV}"
+
+LICENSE="BSD-2"
+SLOT="0"
+
+IUSE="+hmac"
+
+DEPEND="
+	www-nginx/ngx_devel_kit
+	hmac? ( dev-libs/openssl )
+"
+RDEPEND="${DEPEND}"
+
+src_prepare() {
+	sed -E -i 's/USE_OPENSSL|MAIL_SSL/GENTOO_USE_HMAC/g' \
```

Comment:
Can you add comments for these seds please explaining what they're doing and why? Or turn them into a patch if possible/appropriate?